### PR TITLE
Enabling XUNIT testcases for storageadmin powershell, Fix Get-ACSStorageAccount by AccountId

### PR DESCRIPTION
**Pull request by cheguv**
This checklist is used to make sure that common issues in a pull request are covered by the creator. You can find a more complete discussion of PowerShell cmdlet best practices [here](https://msdn.microsoft.com/en-us/library/dd878270%28v=vs.85%29.aspx).

Below in **Overall Changes**, check off the boxes that apply to your PR. Within each of the categories that you select, make sure that you can check off **all** of the boxes.
## Overall Changes
- [x] [**MANDATORY** - General changes](#general)
### General
- [x] Title of the PR is clear and informative
- [x] There are a small number of commits that each have an informative message
- [x] If it applies, references the bug/issue that the PR fixes
- [x] All files have the Microsoft copyright header
- [x] Cmdlets refer to management libraries through nuget references - no dlls are checked in
- [x] The PR does not introduce breaking changes (unless a major version change occurs in the assembly and module)

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@cheguv__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (cheguv). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: AzurePowershell.Test.targets [hunk 1]
```diff
@@ -84,6 +84,7 @@
       <XUnitTests Include=".\src\ResourceManager\AzureBackup\Commands.AzureBackup.Test\bin\Debug\Microsoft.Azure.Commands.AzureBackup.Test.dll"/>
       <XUnitTests Include=".\src\ResourceManager\NotificationHubs\Commands.NotificationHubs.Test\bin\Debug\Microsoft.Azure.Commands.NotificationHubs.Test.dll"/>
       <XUnitTests Include=".\src\Common\Commands.Common.Authentication.Test\bin\Debug\Microsoft.Azure.Commands.Common.Authentication.Test.dll"/>
+	  <XUnitTests Include=".\src\ResourceManager\AzureStackStorage\Commands.AzureStackStorage.Tests\bin\Debug\Microsoft.AzureStack.Commands.StorageAdmin.Test.dll"/>
```
Comment (markcowl) [inline AzurePowershell.Test.targets]: @cheguv fix spacing and also, please pull the latest changes

**Comment by cheguv** [comment]
Hi Mark, Fixed spacing and also merged changes from upstream. Can you review the changes and accept the pull request.

**Comment by markcowl** [comment]
@azuresdkci add to whitelist

**Comment by markcowl** [comment]
on demand run here: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/1128/

**Comment by markcowl** [comment]
@cheguv Tests are failing

**Comment by cheguv** [comment]
Mark, Can you please take the pull request. All tests are passing.

**Comment by cormacpayne** [comment]
@cheguv would you mind pulling from the release-1.2.4 branch to update it with the latest changes?

Also, can you please fill out the checkbox at the top of the pull request? It helps us and the contributor to ensure that the pull request is following the guidelines that we have set.

**Comment by markcowl** [comment]
@cheguv The PR is out of date, please rebase on the latest commit in the release-1.2.4 branch

**Comment by cheguv** [comment]
Rebased the PR, filled out the mandatory checklist, updated title to better reflect the changes.

**Comment by cormacpayne** [comment]
On-demand: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/1140/

**Pull request merged**